.reek

```diff
@@ -43,6 +43,7 @@ TooManyStatements:
   max_statements: 6
   exclude:
     - Users::PhoneConfirmationController
+    - OpenidConnect::AuthorizationController
```

Comment:
If I had to choose between too many statements and too many methods/class too long, I would prefer smaller methods and then later take a look at the class and see if it really is doing too much.